### PR TITLE
Added couple of DirectX resources

### PR DESCRIPTION
**Pull request by tiredbunny**
I added some DirectX resources, namely, 3Dgep and bell0bytes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,26 +17,27 @@
...
-		
+
 		## APIs
-		
+
 		### Direct3D
 		* [DirectX Specification](https://microsoft.github.io/DirectX-Specs/)
-		
+		* [3DGEP blog - DirectX Tutorials](https://www.3dgep.com/category/graphics-programming/directx/)
+		* [bell0bytes blog - DirectX Fundamentals](https://bell0bytes.eu/direct-x-com/)
```
Comment (nickclark2016) [inline index.html]: Make it all alphabetical please

**Review by nickclark2016 — CHANGES_REQUESTED** [review]

**Review by nickclark2016 — APPROVED** [review]

**Comment by DethRaid** [comment]
Probably worth adding MSDN https://docs.microsoft.com/en-us/windows/win32/api/_direct3d12/ to the list of D3D resoruces

**Review by Pitj3 — APPROVED** [review]
Yep, this is fine

**Comment by tiredbunny** [comment]
> Probably worth adding MSDN https://docs.microsoft.com/en-us/windows/win32/api/_direct3d12/ to the list of D3D resoruces

I think MSDN is easily searchable by Google (and other search engines); we should keep the list limited to hard to find resources.

**Pull request merged**